src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1109,6 +1159,8 @@ private static String addPrefix(String server) {
      * See http://msdn.microsoft.com/en-us/library/aa746475(VS.85).aspx for the syntax by example.
      * WANTED: The specification of the syntax.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", 
+            justification = "Filter may be changed in the runtime by scripts if required")
```

Comment:
I think this should just be made `final`. The reason it was non-final in the past was because the value was not exposed in the UI. A while back we switched to have the filter configurable via the UI, so no need to have this value live-modifiable any more.